.gitignore

```diff
@@ -1,3 +1,5 @@
 node_modules
 tmp
 git_repo_local_cache/
+
+*.swp
```

Comment:
We shouldn't update `.gitignore` in this PR, it's not related. I also suggest to include it in your personal global gitignore list, like:
https://github.com/PeterDaveHello/Unitial/blob/master/gitignore_global#L2
https://github.com/PeterDaveHello/Unitial/blob/master/gitconfig#L72